test/e2e/testsuites/testsuites.go

```diff
@@ -1033,15 +1034,26 @@ func SetNodeLabels(c clientset.Interface, node *v1.Node, newLabels map[string]st
 		node.Labels = originalLabels
 		_, _ = c.CoreV1().Nodes().Update(context.Background(), node, metav1.UpdateOptions{})
```

Comment:
We probably should add retry on conflict with clean up function as well.

Reply:
Yeah makes sense.